src/test/java/bio/terra/janitor/integration/TrackResourceIntegrationTest.java

```diff
@@ -108,6 +108,28 @@ public void subscribeAndCleanupResource_googleBucket() throws Exception {
     assertNull(storageCow.get(blobId));
   }
 
+  @Test
+  public void subscribeAndCleanupResource_googleBlob() throws Exception {
```

Comment:
probably should also have a test for deleting a blob that's already been deleted, or when the bucket's already deleted. WDYT?

Reply:
added a new one